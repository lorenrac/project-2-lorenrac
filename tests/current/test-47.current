ERROR (line 3): Unexpected token 'ID'
