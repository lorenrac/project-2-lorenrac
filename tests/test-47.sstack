// ERROR: Mis-placed ELSE
VAR count = "aaaa"
WHILE (count) count = count - "a"
ELSE PRINT "Illegal ELSE"
