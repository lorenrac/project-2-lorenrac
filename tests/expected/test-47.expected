ERROR (line 4): Unexpected token 'ELSE'
